src/commands/title-escrow/changeHolder.ts

```diff
@@ -0,0 +1,58 @@
+import { Argv } from "yargs";
+import { error, info, success, warn } from "signale";
+import { getLogger } from "../../logger";
+import { changeHolderOfTitleEscrow } from "../../implementations/title-escrow/changeHolder";
+import { TitleEscrowChangeHolderCommand } from "./title-escrow-command.type";
+import { withGasPriceOption, withNetworkAndWalletSignerOption } from "../shared";
+import { getEtherscanAddress } from "../../utils";
+
+const { trace } = getLogger("title-escrow:change-holder");
+
+export const command = "change-holder [options]";
+
+export const describe = "Changes the holder of the transferable record to another address";
+
+export const builder = (yargs: Argv): Argv =>
+  withGasPriceOption(
+    withNetworkAndWalletSignerOption(
+      yargs
+        .option("address", {
```

Comment:
might be less confusing if the flag is specifically named token-registry-address , since we are dealing with both title escrow contracts and token registries here

Reply:
👌